### ISSUE TITLE
Type Optional[List[str]] rejects all requests

### ISSUE THREAD

**Issue by jramseygreen** [Ge0rg3/flask-parameter-validation#31]
When setting a parameter type to Optional[List[str]], whether I send it in the payload or not it will return error response "Parameter must be type 'typing.Optional[typing.List[str]]'"

**Comment by jramseygreen** [comment]
line 167 of parameter_validation.py

```py
# In python3.7+, typing.Optional is used instead of typing.Union[..., None]
        if expected_input_type_str.startswith("typing.Optional"):
            new_type = expected_input_type.__args__[0]
            expected_input_type = new_type
```

Change to

```py
        # In python3.7+, typing.Optional is used instead of typing.Union[..., None]
        if expected_input_type_str.startswith("typing.Optional"):
            new_type = expected_input_type.__args__[0]
            expected_input_type = new_type
            expected_input_type_str = str(new_type)
```

fixes the issue

**Comment by Ge0rg3** [comment]
Closed by #33, thanks again!
